app/src/main/java/com/sdp/movemeet/view/navigation/Navigation.java

```diff
@@ -15,41 +36,190 @@
 
 public class Navigation extends AppCompatActivity {
```

Comment:
I suppose you haven't worked on the test for this new Navigation class. It could be nice to do it, to higher up the coverage aswell.

Reply:
No indeed, I've spent way too much time trying to solve other errors and haven't had a chance to look at the tests for this class yet.